.ci/build

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
```

Comment:
Can you please change the copyright date to 2020?

Reply:
done